code/game/objects/items/storage/boxes.dm

```diff
@@ -1143,3 +1143,60 @@
 		/obj/item/stock_parts/matter_bin = 2,
 		/obj/item/screwdriver = 1)
 	generate_items_inside(items_inside,src)
+
+/obj/item/storage/box/material
```

Comment:
We already have this under closet/syndicate/materials

Reply:
You mean closet/syndicate/resources?
Closet is closet, box is a box, can't spawn closet in backpack too.